### ISSUE TITLE
Inactive element included in focus order (2.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-modals#94]
### Issue from RE NXT accessibility audit 2019
**Avoid placing inactive elements in the focus order**
The div element below is added to the tab order of the page using tabindex="0"

`<div _ngcontent-c20="" class="sky-modal-content sky-padding-even-large" role="region" tabindex="0" aria-labelledby="sky-modal-header-id-1" id="sky-modal-content-id-1">`

### Impact
When inactive elements are added to the tab order of the page, it creates confusion.

### Recommedation
Remove unnecessary use of tabindex="0" on inactive elements such as a Div.

Need to find a different solution for when contents of modal needs to be able to scroll via keyboard.
